cmd/agent/dist/conf.d/go_expvar.d/agent_stats.yaml.example

```diff
@@ -144,3 +144,15 @@ instances:
         type: rate
       - path: splitter/PayloadDrops
         type: rate
+
+      # datadog-agent logs-agent monitoring
+      - path: logs-agent/IsRunning
```

Comment:
👍 this'll work indeed, the go_expvar check casts the values to floats, so here it will send `0.` or `1.` for the metric `datadog.agent.logs_agent.is_running`